README.md

```diff
@@ -136,6 +135,20 @@ duplicate([1,2,3,4,5]); // [1,2,3,4,5,1,2,3,4,5]
 * Explain what a single page app is and how to make one SEO-friendly.
 * What is the extent of your experience with Promises and/or their polyfills?
 * What are the pros and cons of using Promises instead of callbacks?
+* What are some of the advantages/disadvantages of writing JavaScript code in a language that compiles to JavaScript?
+
+#### Testing Questions:
+
+* What are some advantages/disadvantages to testing your code?
+* What tools would you use to test your code's functionality?
+* What is the difference between a unit test and a functional/integration test?
+* What is the purpose of a code style linting tool?
```

Comment:
Braindump but maybe we should have some more code quality questions (not for this PR though)